zap/src/main/java/org/zaproxy/zap/extension/pscan/PassiveScanTask.java

```diff
@@ -204,16 +204,18 @@ public void run() {
                 }
             } else {
                 logger.error(
-                        "Parser failed on record {} from History table", href.getHistoryId(), e);
+                        "Parser failed on record {} from History table (perhaps it was deleted between when it was enqueued and now)",
+                        href.getHistoryId());
+                logger.debug(e);
```

Comment:
This will just log the exception's message not the stacktrace. Was that on purpose?

Reply:
Nope, I can adjust it